### PR TITLE
Adjustment

### PR DESCRIPTION
**Pull request by jburel**
Update Readme, add classifiers, travis, file to ignore etc.

cc @joshmoore

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+============
+
+* OMERO 5.4.0 or newer
+* Python 2.6+
+
+
+Installing from PyPI
+====================
+
+This section assumes that an OMERO.web is already installed.
```
Comment (joshmoore) [inline README.rst]: s/web/py/ ?

File: setup.py [hunk 2]
```diff
@@ -1,18 +1,63 @@
-from distribute_setup import use_setuptools
-use_setuptools()
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 University of Dundee.
```
Comment (joshmoore) [inline setup.py]: year

File: setup.py [hunk 3]
```diff
@@ -1,18 +1,63 @@
...
 
+def read(fname):
+    """
+    Utility function to read the README file.
+    :rtype : String
+    """
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+
+
+version = '0.0.1'
```
Comment (joshmoore) [inline setup.py]: shall we start with 0.1.0 i.e. not completely alpha?

**Comment by joshmoore** [comment]
Few minor issues and then I'd say let's get this in and I'll start propagating it. I'm not sure about the impact of not using the newer `distribute_setup.py`. That may be something we want to roll out to all repositories in a follow-up step.

**Comment by joshmoore** [comment]
Looks good & green. I'm going to go ahead and merge and use this on the other repos. Thanks, @jburel.

**Comment by jburel** [comment]
I will follow the same pattern for omero-cli-render now

**Pull request merged**